### ISSUE TITLE
[Debug] react-native@ isn't supported

### ISSUE THREAD

**Issue by javiereotero** [microsoft/vscode-react-native#1094]
First, I wanted to thank the team of contributors for authoring such a valuable tool for the React Native community. I recently switched to VSCode and absolutely love the experience so far. When looking to bring my debug game to the next level with this extension, I unfortunately ran into the following. Thank you in advance for any insights you can provide.

## Actual Behavior
(pulling in some Actual Behavior steps from [this issue](https://github.com/microsoft/vscode-react-native/issues/1070), although the resolution from that issue didn't correct my scenario.)
1. When adding configuration through VSCode Debug menu settings, I select either "Attach to packager" or "Debug in Exponent" to generate launch.json file.
2. Clicked "green arrow" or F5 to start debugging.
3. The debugger float dialog appeared for few seconds then disappeared.
4. App was never started.
5. Output window shows several messages, including "[Debug] react-native@ isn't supported"

## Relevant Environment Information
1. I have the codebase folder opened in VSCode (it is not a Workspace)
2. I can see this issue occur whether I use my project or create a project using `expo init my-new-project`
3. I have added the following to my VSCode User Settings:
```
"react-native-tools": {
        "packager": {
            "port": 19001
        },
        "showInternalLogs": true,
        "logLevel": "Trace"
    }
```
4. I have attempted to use both "Attach to package" and "Debug in Exponent". I've also played with including additional settings like port, sourceMaps, and trace. Below are two screenshots of the configuration and output.
<img width="1249" alt="Screen Shot 2019-09-03 at 10 27 30 AM" src="https://user-images.githubusercontent.com/764211/64189888-cfb9dc00-ce3a-11e9-85be-3349793a0255.png">
<img width="1046" alt="Screen Shot 2019-09-03 at 10 30 26 AM" src="https://user-images.githubusercontent.com/764211/64189890-cfb9dc00-ce3a-11e9-864a-daa82bee4cb5.png">
5. As an aside, I'm able to connect using _React Native Debugger_ recommended by the Expo Documentation. The biggest downside is no support for breakpoints in VSCode.
6. My package.json has the "sdkVersion" set to "34.0.0"

## Expected Behavior
1. App and debugger should start.
**or**
1. Attachment to packager should succeed.

## Software versions
- React-Native VS Code extension version: 0.11.0
- VSCode version: 1.37.1
- OS platform and version: macOS Mojave 10.14.5
- NodeJS version: 12.8.0
- React Native version: 0.59.8
- Expo SDK version (include if relevant): 34.0.0

## Outputs (Include if relevant)
**- Output of the Debug Console (View -> Toggle Debug Console):**
```
Debug Console is empty (no output to add here)
```
**- Output of the React-Native output channel (View -> Toggle Output -> Select React-Native in ListBox):**
```
[Debug] Begin to activate...

[Debug] Extension version: 0.11.0

[Debug] Projects found: 1

[Debug] Add project: REDACTED

[Debug] React Native version: 

[Debug] react-native@ isn't supported
```
**- Output of the Developer Tools console (Help -> Toggle Developer Tools -> Select Console tab):**
```
[Extension Host] debugger listening on port 32883
Empty otherwise
```

**Comment by StanS-AWS** [comment]
Are you able to see the version of react-native in vsc terminal? 
PS > react-native -v
react-native-cli: 2.0.1
react-native: 0.59.8

I was able to fix similar issue when expo-cli was installed but react-native-cli did not. The fix was to run: 
npm install -g react-native-cli

**Comment by RedMickey** [comment]
Hi, @javiereotero, thanks for reaching us. Was the previous advice useful? In our case it helped to solve this issue. We found the reason of this behavior and now we are investigating how to fix it. We created the issue [`Add local React Native CLI support #1101`](https://github.com/microsoft/vscode-react-native/issues/1101), which contains your problem, and added it to backlog. Therefore, we close this issue. You can track progress in that issue [`Add local React Native CLI support #1101`](https://github.com/microsoft/vscode-react-native/issues/1101). Feel free to reopen if needed.

**Comment by javiereotero** [comment]
Hi @StanS-AWS and @RedMickey, I believe the previous advice _may_ have been useful. I ran `react-native -v` several times and got conflicting results in/out of the directory. When installing `npm install -g react-native-cli` I ran into a few npm related issues that I need to resolve before verifying this fix. Once I do that, I'll post my results here just to confirm it resolves the issue.

Really appreciate all your support!
